src/app/news/news-list.component.ts

```diff
@@ -82,6 +87,25 @@ export class NewsListComponent implements OnChanges {
       this.dialogsFormService.closeDialogsForm();
       this.dialogsLoadingService.stop();
     });
+    this.sendNewsNotifications(oldNews);
+  }
+
+  sendNewsNotifications(oldNews) {
+    const userId = this.userService.get()._id;
```

Comment:
This is not the correct user to send the notification to.  It should be sent to the author of the message that was replied to, not the currently logged in user.